frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -106,6 +144,23 @@ const IconButton = styled(MuiIconButton)({
   },
 });
 
+interface resultOption {
+  id: string;
+  label: string;
+}
+
+interface ResultProps {
+  option: resultOption;
```

Comment:
Why have `option` here? It seems like the result can have an id and label props directly

Reply:
yeah we could do that let me change